### PR TITLE
handle empty attachments array for bubbles

### PR DESCRIPTION
**Pull request by jonjonray**
**Overview:**
We were adding the attachments containers in AnnouncementBubble and MessagingBubble even when the attachments array was empty. This wasn't affecting the messaging bubble size since the container was of size 0 but the AnnouncementBubble attachment container had some margin on the top, which was creating some extra whitespace.

**Screenshots/GIFs:**
BEFORE
Screen Shot 2021-06-22 at 3 05 54 PM

AFTER
Screen Shot 2021-06-22 at 3 06 13 PM

**Testing:**

- [ ] Unit tests
- Manual tests:
  - [ ] Chrome
  - [ ] Safari
  - [ ] IE11

**Roll Out:**

- Before merging:
  - [ ] Updated docs
  - [ ] Bumped version in `package.json`
    - Breaking change?
      - If it is a beta component run `npm version minor`
      - If the component is not in beta run `npm version major`
    - New component or backward-compatible component feature change? Run `npm version minor`
    - Only changing documentation? All good. Skip this step.
  - After creating a new component, make sure to add it to the Components List in `ComponentsView.jsx`. To do so:
    - [ ] Add a screenshot of the component in `docs/assets/img` with the format `<COMPONENT URL LINK>.png`
- After merging:
  - [ ] Deployed updated docs (`make deploy-docs`)
  - [ ] Posted in #eng if I made a breaking change to a beta component

### CODE REVIEW

File: src/MessagingAttachment/MessagingAttachment.less [hunk 1]
```diff
@@ -22,6 +22,7 @@
   height: @ICON_REGULAR_SIZE;
   border: @size_4xs solid @neutral_silver;
   border-radius: @size_2xs;
+  box-sizing: content-box;
```
Comment (jonjonray) [inline src/MessagingAttachment/MessagingAttachment.less:25]: Sneaking in this one which is unrelated to the whitespace issue

File: src/AnnouncementBubble/NormalAnnouncementBubble.tsx [hunk 2]
```diff
@@ -59,7 +59,9 @@ export const NormalAnnouncementBubble: React.FC<Props> = ({
       <Linkify componentDecorator={componentDecorator} matchDecorator={matchDecorator}>
         <div className={cssClass("messageBody")}>{children}</div>
       </Linkify>
-      {attachments && <FlexBox className={cssClass("attachmentContainer")}>{attachments}</FlexBox>}
+      {attachments?.length > 0 && (
```
Comment (clcaelynn-clever) [inline src/AnnouncementBubble/NormalAnnouncementBubble.tsx:62]: Ah yes, the classic `empty array isn't falsey`. 😄

File: src/MessagingAttachment/MessagingAttachment.less (hunk 1 above)
Comment (clcaelynn-clever) [reply src/MessagingAttachment/MessagingAttachment.less:25]: Nice nice

**Review by clcaelynn-clever — APPROVED** [review]
LGTM! 🚀
